src/hooks/useImgix.ts

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { Source } from 'react-native-fast-image';
+
+import { maybeSignSource, maybeSignUri } from '../handlers/imgix';
+
+export function useSafeImageUri(
+  maybeUnsafeUri: string | undefined
+): string | undefined {
+  return React.useMemo(() => {
+    return maybeSignUri(maybeUnsafeUri);
+  }, [maybeUnsafeUri]);
+}
+
+export default function useImgix(source: Source): Source {
```

Comment:
Doesn't look like this particular default function is used currently, but we're keeping it for potential future use?

Reply:
Hey Jin! Amazing catch 💪, this should 100% be used by `ImgixImage.ts`. D'oh!

```typescript
class ImgixImage extends React.PureComponent<ImgixImageProps> {
  render() {
    // Why am I not doing anything with the source prop?! 🙈
    const { Component: maybeComponent, source, ...props } = this.props;
    const Component = maybeComponent || FastImage;
    return <Component {...props} source={source} />;
  }
}
```

I think I had to remove the hook when I migrated over to using a class component, somewhere during testing it never made it back in. I am a fool.

Even after the update, you're right, this function is unused... I see no harm in removing it since it doesn't look like there's any existing use case for it. I'll fix this ASAP.